README.md

```diff
@@ -180,7 +179,7 @@ Some of the Trillian Go code is autogenerated from other files:
    [stringer](https://godoc.org/golang.org/x/tools/cmd/stringer) tool (`go get
    golang.org/x/tools/cmd/stringer`).
 
-Re-generating mock or protobuffer files is only needed if you're changing
+Re-generating mock or protobuffer files are only needed if you're changing
```

Comment:
This was correct originally - please revert (the "is" relates to the act of doing, rather than the plurality of files themselves.)